dbt-tests-adapter/dbt/tests/adapter/incremental/test_incremental_on_schema_change.py

```diff
@@ -84,3 +84,31 @@ def test_run_incremental_fail_on_schema_change(self, project):
 
 class TestIncrementalOnSchemaChange(BaseIncrementalOnSchemaChange):
     pass
+
+
+class BaseIncrementalCaseSenstivityOnSchemaChange:
+    @pytest.fixture(scope="class")
+    def models(self):
+        return {
+            "src_artists.sql": fixtures._MODELS__SRC_ARTISTS,
+            "dim_artists.sql": fixtures._MODELS__DIM_ARTISTS,
+        }
+
+    def test_run_incremental_check_quoting_on_new_columns(self, project):
+        select = "src_artists dim_artists"
+        run_dbt(["run", "--models", select, "--full-refresh"])
+        run_dbt(["show", "--inline", "select * from {{ ref('dim_artists') }}"])
+        res, logs = run_dbt_and_capture(
+            ["show", "--inline", "select * from {{ ref('dim_artists') }}"]
+        )
+        assert "Job" not in logs
+        run_dbt(["run", "--vars", "{'version': 1}"])
+        run_dbt(["show", "--inline", "select * from {{ ref('dim_artists') }}"])
+        res, logs = run_dbt_and_capture(
+            ["show", "--inline", "select * from {{ ref('dim_artists') }}"]
+        )
+        assert "Job" in logs
+
+
+class TestIncrementalCaseSenstivityOnSchemaChange(BaseIncrementalCaseSenstivityOnSchemaChange):
```

Comment:
We don't need this class. The reason `TestIncrementalOnSchemaChange` is there is to preserve backwards compatibility. These tests used to run when testing `dbt-postgres`. Now that that's in its own repo and explicitly implements tests from `dbt-tests-adapter`, we no longer need to create these classes here.